README.md

```diff
@@ -43,6 +43,12 @@ For detailed information on how to build and run the samples, check out our `REA
 [sentry-samples](https://github.com/getsentry/sentry-kotlin-multiplatform/tree/main/sentry-samples)
 folder.
 
+## Apple Privacy Manifest
+
+Starting with [May 1st 2024](https://developer.apple.com/news/?id=3d8a9yyh), iOS apps are required to declare approved reasons to access certain APIs. This also includes third-party SDKs.
+If you are using the Sentry Kotlin Multiplatform SDK with Apple device targets then update your Sentry Cocoa SDK to `>=8.25.0`.
+For more information, please refer to our [Apple Privacy Manifest Guide](https://docs.sentry.io/platforms/kotlin-multiplatform/data-management/apple-privacy-manifest/).
```

Comment:
```suggestion
Starting with [May 1st 2024](https://developer.apple.com/news/?id=3d8a9yyh), apps submitted to the Apple App Store are required to declare approved reasons to access certain privacy-relevant APIs. This also includes usages of these APIs via third-party SDKs. To ensure compliance, update your Sentry Cocoa SDK to `>=8.21.0`.
For more information, refer to our [Apple Privacy Manifest Guide](https://docs.sentry.io/platforms/kotlin-multiplatform/data-management/apple-privacy-manifest/).
```

Reply:
reasons for my suggestions: it's not just iOS apps, but should also apply to macOS, tvOS, etc. also it's not so much about targeting Apple devices, but submitting to the Apple App Store